apple/swift-docc
# Use external JS/CSS assets for custom header/footer in docs

Bug/issue #, if applicable: 84904657

## Summary

To support server environments with a stricter `Content-Security-Policy` header, the CSS and JavaScript assets have been extracted from inline `<style>`/`<script>` tags to externally referenced resources.

## Testing

Steps:
1. Configure `DOCC_HTML_DIR` env var or set it as part of the following commands
2. Run `bin/preview-docs SwiftDocC` and verify that the header/footer look the same as before
3. Run `bin/preview-docs SwiftDocCUtilities` and verify that the header/footer look the same as before
4. Run `bin/preview-docs DocC` and verify that the header/footer look the same as before

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] Added tests (none applicable)
- [ ] Updated documentation if necessary (none applicable)
